### PR TITLE
Motions 2019 02 cwg 2

### LINKED ISSUE #2682: [2019-02 CWG Motion 2] P1359R0 Core Language Working Group "tentatively ready" Issues (Kona 2019)
[P1359R0](https://wg21.link/p1359r0)

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #2682.

FYI: CWG2083 Also fixes CWG2103 and CWG2170.

### CODE REVIEW

**Comment by jensmaurer** [comment]
@burblebee: "All checks have failed".

File: source/basic.tex [hunk 1]
```diff
@@ -290,12 +292,17 @@
 \item If \tcode{e} is a subscripting operation\iref{expr.sub} with
 an array operand, the set contains the potential results of that operand.
 \item If \tcode{e} is a class member access
-expression\iref{expr.ref}, the set contains the potential results of
-the object expression.
+expression\iref{expr.ref} of the form
+\tcode{e1 . template\opt{} e2}
```
Comment (cpplearner) [inline source/basic.tex]:
```suggestion
\tcode{e1 . \opt{template} e2}
```

**Comment by burblebee** [comment]
> @burblebee: "All checks have failed".

I rebased on master and it introduced some Overfull and undefined references.  @zygoloid master is currently broken and won't build!  I will try to find a good git hash in master and rebase on that, but it will mean having to force push the branch.

File: source/basic.tex (hunk 1 above)
Comment (burblebee) [reply source/basic.tex]: thanks!

**Comment by zygoloid** [comment]
Force-pushed with fixes:

CWG1938: Expanded on implementation-defined behavior index entry
CWG2020: P1359R0 missed two edits from the version discussed in CWG (confirmed with Mike), missing formatting for id-expression
CWG2051: Add an "or" before the last bullet
CWG2266: The object expression cannot *be* the current instantiation (that's a name, not an expression)
CWG2380: Inconsistent punctuation after "error"
CWG2381: "noexcept" missing code font
CWG2386: "value" missing code font

**Pull request merged**